File: src/generator/vue-admin/crawler/crawler/src/main/java/com/jasmine/crawler/pojo/entity/SiteBlockRule.java
/*----------------------------------------------------------------------------
 * Jasmine code generator, a tool to build web crud application,with spring-
 * boot, mybatis, mysql,swagger,spring-security.
 * Generated at 2020-7-28 4:23:10 PM 
 * All rights reserved by fuanlei(email:767550758@qq.com) since 2019
 *---------------------------------------------------------------------------*/
package com.jasmine.crawler.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("site-url-block-rule")
@Getter
@Setter
public class SiteBlockRule  {

    @ApiModelProperty("")
    private Integer id;

    @ApiModelProperty("description of the rul")
    private String description;

    @ApiModelProperty("site the rule belong to")
    private Integer siteId;

    @ApiModelProperty("http status code")
    private Integer httpStatus;

    @ApiModelProperty("checkTyp: json array")
    private Integer checkType;

    @ApiModelProperty("json array of string to match")
    private String keywords;

    @ApiModelProperty("if match the page result returned")
    private Integer pageResultType;

    @ApiModelProperty("enable status")
    private Integer enableStatus;

    @ApiModelProperty("")
    private Date createTime;

}

File: src/generator/vue-admin/crawler/db/siteBlockRule.js
let siteBlockRule = {
	name: "siteBlockRule",
	description: "site-url-block-rule",
	columns: {
		id: {
			name: "id",
			description: "",
			isPk: true,
			autoIncrement: true,
			rawName: "id",
			nullable: false,
			type: {
				name: "int",
				length: 11
			}
		},
		description: {
			name: "description",
			description: "description of the rul",
			rawName: "description",
			nullable: true,
			type: {
				name: "varchar",
				length: 200
			}
		},
		siteId: {
			name: "siteId",
			description: "site the rule belong to",
			rawName: "site_id",
			nullable: false,
			type: {
				name: "int",
				length: 11
			}
		},
		httpStatus: {
			name: "httpStatus",
			description: "http status code",
			rawName: "http_status",
			nullable: true,
			type: {
				name: "int",
				length: 11
			}
		},
		checkType: {
			name: "checkType",
			description: "checkTyp: json array",
			rawName: "check_type",
			nullable: false,
			type: {
				name: "smallint",
				length: 6
			}
		},
		keywords: {
			name: "keywords",
			description: "json array of string to match",
			rawName: "keywords",
			nullable: false,
			type: {
				name: "varchar",
				length: 200
			}
		},
		pageResultType: {
			name: "pageResultType",
			description: "if match the page result returned",
			rawName: "page_result_type",
			nullable: false,
			type: {
				name: "smallint",
				length: 6
			}
		},
		enableStatus: {
			name: "enableStatus",
			description: "enable status",
			defaultValue: "enable status",
			rawName: "enable_status",
			nullable: true,
			type: {
				name: "smallint",
				length: 6
			}
		},
		createTime: {
			name: "createTime",
			description: "",
			defaultValue: "",
			rawName: "create_time",
			nullable: false,
			type: {
				name: "timestamp"
			}
		}
	},
	rawName: "site_block_rule",
	primaryColumn: "id",
	nameColumn: {
		name: "siteId",
		score: 1.203701901893523,
		content: "site_id",
		weight: 1,
		totalTokenCount: 2
	}
};

let columnsArray = [
	"id",
	"description",
	"siteId",
	"httpStatus",
	"checkType",
	"keywords",
	"pageResultType",
	"enableStatus",
	"createTime"
];

function create() {
  return {};
}

siteBlockRule.columnsArray=columnsArray;
siteBlockRule.create=create;
module.exports.siteBlockRule=siteBlockRule;